Drivers/Inc/Display.h

```diff
@@ -4,7 +4,7 @@
  * @brief Function prototypes for the display driver.
  * 
  * This contains function prototypes relevant to sending/receiving messages
- * to/from our Nextion HMI. Call assertDisplayError after calling any of the
+ * to/from our Nextion HMI. Call assertUpdateDisplayError after calling any of the
```

Comment:
I believe this is not true. We only call assertUpdateDisplayError if something offers us an UpdateDisplayError_t. Otherwise, any driver errors that are returned should be handled by the callee.

Reply:
I see, thanks for clarifying that!